Assets/SEE/XR/XRSEEActions.cs

```diff
@@ -0,0 +1,218 @@
+    /// <summary>
+    /// Shows the source name of the hovered or selected object as a text label above the
+    /// object. In between that label and the game object, a connecting bar
+    /// will be shown.
+    /// </summary>
+    private ShowLabel showLabel;
+
+    // Awake is always called before any Start functions.
+    private void Awake()
+    {
+        tooltip.action.performed += Tooltip;
+        undo.action.performed += Undo;
+        redo.action.performed += Redo;
+        inputAction.action.Enable();
+        inputAction.action.performed += Action;
+        RayInteractor = m_RayInteractor;
+        Selected = false;
+    }
+    /// <summary>
+    /// This method gets called, when the user begins to hover over an interactable.
+    /// </summary>
+    /// <param name="args">Event data associated with the event when an Interactor first initiates hovering over an Interactable.</param>
+    public void OnHoverEnter(HoverEnterEventArgs args)
+    {
+        hovering = true;
+        hoveredGameObject = args.interactableObject.transform.gameObject;
+        if (GlobalActionHistory.Current() == ActionStateTypes.Move)
+        {
+            oldParent = args.interactableObject.transform.parent;
+        }
+        hoveredGameObject.transform.TryGetComponent(out showLabel);
+        showLabel?.On();
+        if (hoveredGameObject.transform.TryGetComponent(out InteractableObject io))
+        {
+            io.SetHoverFlag(HoverFlag.World, true, true);
+        }
+    }
+    /// <summary>
+    /// This method gets called, when the user stops hovering over an interactable.
+    /// </summary>
+    /// <param name="args">Event data associated with the event when an Interactor stops hovering over an Interactable.</param>
+    public void OnHoverExited(HoverExitEventArgs args)
+    {
+        hovering = false;
+        hoveredGameObject = args.interactableObject.transform.gameObject;
+        hoveredGameObject.transform.TryGetComponent(out showLabel);
+        showLabel?.Off();
+        if (hoveredGameObject.transform.TryGetComponent(out InteractableObject io))
+        {
+            io.SetHoverFlag(HoverFlag.World, false, true);
+        }
+    }
+    /// <summary>
+    /// Is true, when the button for the primary actions is pressed.
+    /// </summary>
+    public static bool Selected { get; set; }
+
+    public static bool SelectedFlag { get; set; }
+    /// <summary>
+    /// The GameObject, which should be rotated.
+    /// </summary>
+    public static GameObject RotateObject { get; set; }
+    /// <summary>
+    /// This method gets called, when the button for the primary actions is pressed.
+    /// </summary>
+    /// <param name="context">Information provided to action callbacks about what triggered an action.</param>
+    private void Action(InputAction.CallbackContext context)
+    {
+        if (hovering)
+        {
+            if (GlobalActionHistory.Current() == ActionStateTypes.Move && Selected == true)
```

Comment:
A boolean need not be compared to `true`:
```suggestion
            if (GlobalActionHistory.Current() == ActionStateTypes.Move && Selected)
```